app/frontend/src/features/auth/signup/AccountForm.tsx

```diff
@@ -291,7 +291,7 @@ export default function AccountForm() {
                 row
                 aria-label="gender"
                 name="gender-radio"
-                onChange={onChange}
+                onChange={(e, value) => onChange(e, value)}
```

Comment:
How is this different to what it was before? I think it'd call the function with those arguments anyway with `onChange={onChange}`?

Reply:
Whoops, it's meant to be `onChange(value)`